x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_match.tsx

```diff
@@ -98,27 +105,72 @@ export const AutocompleteFieldMatchComponent: React.FC<AutocompleteFieldMatchPro
     isLoadingSuggestions,
   ]);
 
-  return (
-    <EuiComboBox
-      placeholder={inputPlaceholder}
-      isDisabled={isDisabled}
-      isLoading={isLoadingState}
-      isClearable={isClearable}
-      options={comboOptions}
-      selectedOptions={selectedComboOptions}
-      onChange={handleValuesChange}
-      singleSelection={{ asPlainText: true }}
-      onSearchChange={onSearchChange}
-      onCreateOption={onChange}
-      isInvalid={!isValid}
-      onFocus={setIsTouchedValue}
-      sortMatchesBy="startsWith"
-      data-test-subj="valuesAutocompleteComboBox matchComboxBox"
-      style={fieldInputWidth ? { width: `${fieldInputWidth}px` } : {}}
-      fullWidth
-      async
-    />
-  );
+  const getDefaultInput = (): JSX.Element => {
+    return (
+      <EuiComboBox
+        placeholder={inputPlaceholder}
+        isDisabled={isDisabled}
+        isLoading={isLoadingState}
+        isClearable={isClearable}
+        options={comboOptions}
+        selectedOptions={selectedComboOptions}
+        onChange={handleValuesChange}
+        singleSelection={{ asPlainText: true }}
+        onSearchChange={handleSearchChange}
+        onCreateOption={onChange}
+        isInvalid={!isValid}
+        onFocus={setIsTouchedValue}
+        sortMatchesBy="startsWith"
+        data-test-subj="valuesAutocompleteComboBox matchComboxBox"
```

Comment:
There's a few more of these floating around outside this change-set, so just a heads up. I'm seeing 8 in total when searching `matchComboxBox` in my ide.

```suggestion
        data-test-subj="valuesAutocompleteComboBox matchComboBox"
```

Reply:
Updated!